pom.xml

```diff
@@ -253,6 +253,19 @@
 					</links>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-failsafe-plugin</artifactId>
+				<version>${maven-failsafe-plugin.version}</version>
+				<executions>
+					<execution>
+						<goals>
+							<goal>integration-test</goal>
+							<goal>verify</goal>
```

Comment:
This will work when someone does `mvn verify`, but because this old repo does integration tests using `surefire`, the [TravisCI config only does `mvn test`](https://github.com/spring-cloud/spring-cloud-gcp/blob/master/.travis.yml#L31-L33), so this test will continue to not run regularly.

Reply:
Good catch! I guess I'll modify TavisCI config.